x/data/server/msg_anchor.go

```diff
@@ -0,0 +1,86 @@
+package server
```

Comment:
should we keep a consistent naming convention to keep this module in line with ecocredit? over there for server impl functions, we named the file after the RPC name, so in this case just `anchor.go`. and then for sdk.Msg implementations, we did `msg_anchor.go`

Reply:
Hmm... I started without `msg_` but thought this might be an improvement and maybe we should update the namespace in the ecocredit module. These are not msgs but they are msg server methods that handle msgs and the prefix allows for better sorting in the directory. We don't use the RPC name for query methods but we still prefix with `query_`.

![Screenshot from 2022-03-30 14-14-02](https://user-images.githubusercontent.com/12519942/160931955-4c186394-d54f-4667-88d9-38e1a907b565.png)


![Screenshot from 2022-03-30 14-18-48](https://user-images.githubusercontent.com/12519942/160932685-63e6a55a-b119-4625-864d-d30f308ed0ca.png)

Reply:
yeah seeing it side by side, i think your suggestion is better. we should open an issue to do the same to eco